src/commands/dev-env-sync-sql.js

```diff
@@ -158,10 +158,23 @@ export class DevEnvSyncSQLCommand {
 			const url = site.homeUrl.replace( /https?:\/\//, '' );
 			if ( ! this.searchReplaceMap[ url ] ) continue;
 
-			this.searchReplaceMap[ url ] = `${ site.blogId }.${ this.landoDomain }`;
+			this.searchReplaceMap[ url ] = `${ this.slugifyDomain( url ) }-${ site.blogId }.${
+				this.landoDomain
+			}`;
 		}
 	}
 
+	slugifyDomain( domain ) {
+		return String( domain )
+			.normalize( 'NFKD' ) // split accented characters into their base characters and diacritical marks
+			.replace( /[\u0300-\u036f]/g, '' ) // remove all the accents, which happen to be all in the \u03xx UNICODE block.
+			.trim() // trim leading or trailing whitespace
+			.toLowerCase() // convert to lowercase
+			.replace( /[^a-z0-9 .-]/g, '' ) // remove non-alphanumeric characters except for spaces, dots, and hyphens
```

Comment:
is there an escape character for ` ` ?

Reply:
Sorry @saroshaga I'm not sure what character is that 😅

Reply:
yeah I'm not sure there's one just for single space, [`\s`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Character_class_escape) will detect all whitespaces